packages/terra-graphs/package.json

```diff
@@ -18,22 +15,114 @@
+    "d3",
+    "d3-module",
+    "graphing"
   ],
   "author": "Cerner Corporation",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/cerner/terra-graphs/issues"
   },
   "homepage": "https://github.com/cerner/terra-graphs#readme",
-  "peerDependencies": {
-    "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+  "files": [
+    "dist",
+    "lib",
+    "README.md"
+  ],
+  "prettier": {
+    "arrowParens": "always",
+    "bracketSpacing": true,
+    "semi": true,
+    "tabWidth": 4,
+    "trailingComma": "none",
+    "overrides": [
+      {
+        "files": "package.json",
+        "options": {
+          "tabWidth": 2,
+          "printWidth": 250
+        }
+      }
+    ]
+  },
+  "publishConfig": {
+    "access": "public"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-theme-context": "^1.0.0"
+    "terra-theme-context": "^1.0.0",
+    "d3": "^5.16.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.10.4",
+    "@babel/plugin-transform-classes": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.10.4",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
+    "@babel/runtime": "^7.10.4",
+    "@cerner/carbon-site-helpers": "^1.11.3",
+    "@cerner/svg-to-carbon": "^1.1.3",
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
+    "@semantic-release/git": "^9.0.0",
+    "autoprefixer": "^9.8.4",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-istanbul": "^6.0.0",
+    "babel-plugin-minify-replace": "^0.5.0",
+    "babel-preset-minify": "^0.5.1",
+    "babel-eslint": "^10.1.0",
+    "commander": "^5.1.0",
+    "conventional-changelog-eslint": "^3.0.8",
+    "css-loader": "^3.6.0",
+    "cssnano": "^4.1.7",
+    "docdash": "^1.2.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsdoc": "^24.0.6",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "express": "^4.17.1",
+    "express-rate-limit": "^5.1.3",
+    "html-webpack-plugin": "^4.3.0",
+    "husky": "^4.2.5",
+    "jasmine-core": "^3.5.0",
+    "jsdoc": "^3.6.4",
+    "karma": "^5.1.0",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage": "^2.0.2",
+    "karma-jasmine": "^3.3.1",
+    "karma-parallel": "^0.3.1",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^4.0.2",
+    "less": "^3.11.2",
+    "less-loader": "^6.2.0",
+    "loader-utils": "^1.4.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "one-cerner-style-icons": "github:cerner/one-cerner-style-icons",
+    "postcss-loader": "^3.0.0",
+    "prettier": "^2.0.5",
+    "puppeteer": "^3.3.0",
+    "rimraf": "^3.0.2",
+    "semantic-release": "^17.1.1",
+    "sinon": "^9.0.2",
+    "style-loader": "^1.2.1",
+    "stylelint": "^13.6.1",
+    "stylelint-config-standard": "^20.0.0",
```

Comment:
I'd maybe push this to the root level to follow the same stylelint standard in the mono repo. It's currently setup to lint for terra's standard: https://github.com/cerner/terra-graphs/blob/main/package.json#L63

Reply:
yeah, it's at root now.